extensions/wikia/Chat2/ChatEntryPoint.class.php

```diff
@@ -29,29 +29,45 @@ static public function onParserFirstCallInit( Parser &$parser ) {
 	 * @return array
 	 */
 	static public function getEntryPointTemplateVars( $isEntryPoint ) {
-		global $wgEnableWallExt, $wgBlankImgUrl;
-		return [
+		global $wgEnableWallExt, $wgBlankImgUrl, $wgUser, $wgSitename;
+		$entryPointGuidelinesMessage = wfMessage( 'chat-entry-point-guidelines' );
+		$chatUsers = ChatEntryPoint::getChatUsersInfo();
+		$chatProfileAvatarUrl = AvatarService::getAvatarUrl( $wgUser->getName(), ChatRailController::AVATAR_SIZE );
+
+		$vars =  [
 			'linkToSpecialChat' => SpecialPage::getTitleFor( "Chat" )->escapeLocalUrl(),
+			'joinTheChatMessage' => wfMsg( 'chat-join-the-chat' ),
```

Comment:
Again, `wfMsg` should not be used here (it is deprecated) - use `wfMessage` instead